File: lib/features/home/domain/entitites/playlist.dart
import 'package:soundspace/features/home/domain/entitites/track.dart';

class Playlist {
  final int id;
  final String title;
  final String image;
  final int follower;
  final String createBy;
  final int trackCount;
  final List<Track> tracks;

  Playlist({
    required this.id,
    required this.follower,
    required this.image,
    required this.title,
    required this.createBy,
    required this.trackCount,
    required this.tracks,
  });
}

File: backend/SoundSpace/Entities/Product/Playlist.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using SoundSpace.Entities.Auth;

namespace SoundSpace.Entities.Product
{
    public class Playlist
    {

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int PlaylistId { get; set; }
        public string Title { get; set; }
        public string Image { get; set; }
        public ICollection<TrackPlaylist> Tracks { get; set; } = new List<TrackPlaylist>();

        public ICollection<PlaylistFollow> Followers { get; set; } = new List<PlaylistFollow>();

        [ForeignKey("User")]
        public int CreateBy { get; set; }
        public User User { get; set; }
    }
}
